onecore_maintenance_extension/models/maintenance.py

```diff
@@ -451,6 +451,11 @@ def write(self, vals):
 
         if vals.get('owner_user_id') or vals.get('user_id'):
             self._add_followers()
+        if vals.get('user_id') and self.stage_id.sequence == 1:
```

Comment:
Nice! This code should probably live in the create function as well if you're assigning a resource while creating a new ticket in odoo.